calc.py

```diff
@@ -1,19 +1,18 @@
-a = float (raw_input ("Vvedite pervoe chislo = ", ))
```

Comment:
It is a good practice to add a sheband line in the beginning of the scripts intended to run with interpreter.